config_example.json

```diff
@@ -15,19 +15,146 @@
         "fennec_nightly_repacks": "/path/to/beetmoverscript/beetmoverscript/templates/fennec_nightly_repacks.yml",
         "fennecx86_nightly": "/path/to/beetmoverscript/beetmoverscript/templates/fennecx86_nightly.yml"
     },
+
     "s3": {
         "firefox": {
-            "bucket": "mozilla-releng-beetmoverscript-firefox",
-            "credentials": {
-                "id": "dummy",
-                "key": "dummy"
+            "nightly": {
+                "push-to-nightly": {
+                    "s3_prefixes": {
+                        "default": {
+                            "s3_prefix": "pub/firefox/nightly/{}-{}/",
```

Comment:
You're right. I should've sticked to these. I started using them initially but then turned to `format` for whatever reasons. Makes sense, thanks!